### ISSUE TITLE
Requires the ability to pause streaming

### ISSUE THREAD

**Issue by NDS7EN** [Genymobile/scrcpy#4893]
Use Ctrl+O to turn off the device screen and still mirror the phone screen on the computer       
        
When the screen is off, the phone can run in the background. But after exiting the mirroring mode, the device screen brightens immediately

Is there a way to turn off the phone screen after passing CTRL+O and continue to run the background script without lighting up the device screen after exiting mirror mode.

Or when you connect to your computer, add a command to pause the transfer of your device's screen, reduce the load on your device to save energy, and run your phone in the background with the screen off

Purpose: Run an automated click script through ADB to make the device run in the background. Turn off the device screen with CTRL+O to reduce heat generation. Use the shortcut key to pause the transmission of device images to stop the resource occupation caused by decoding... to efficiently run equipment foreground tasks

**Comment by NDS7EN** [comment]
no- video does not work. I used scrcpy -m 200 --max-fps=1 to limit the device transmission to reduce the performance consumption of transmission.

**Comment by NDS7EN** [comment]
Currently working fine. My device is cooling down significantly

**Comment by rom1v** [comment]
`scrcpy --no-cleanup` will not restore the display power mode on exit.

**Comment by NDS7EN** [comment]
> `scrcpy --no-cleanup` will not restore the display power mode on exit.

thanks
